Rules.md

```diff
@@ -80,7 +79,7 @@ Token types:
 * 2 x Train troops
     * +2 units in regions with a strategic city, otherwise +1 unit
 * 2 x Get money and invest ("invest" for short)
-    * Receive 2 coins and invest any amount (0-inf) in region
+    * Invest any amount (0-inf) in region
```

Comment:
This makes it almost impossible to invest if your army is the same size as your current investments.